### PR TITLE
Interop CLI

### PR DESCRIPTION
**Pull request by wemeetagain**

### CODE REVIEW

**Comment by mpetrunic** [comment]
Update:
- minimal preset - min 8 validators
- mainnet preset - min 64 validators

**Comment by mpetrunic** [comment]
./bin/lodestar interop -q 1,8 -v 8 -p minimal

**Comment by mpetrunic** [comment]
Status:
- produce attestation has a bug
- produce block fails because deposit tree is never stored on quickstart

File: packages/lodestar/src/chain/stateTransition/util/proposer.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import {hash} from "@chainsafe/ssz";
 
 import {
-  BeaconState,
+  BeaconState, Epoch,
```
Comment (ec2) [inline packages/lodestar/src/chain/stateTransition/util/proposer.ts:8]: Doesn't look like Epoch needs to be imported.

**Review by ec2 — DISMISSED** [review]
Left one comment. Otherwise LGTM

**Pull request closed without merging**